### ISSUE TITLE
indexhot.byte not working in Windows 10

### ISSUE THREAD

**Issue by mjoerussell** [bsansouci/reprocessing-example#12]
I've been trying to start working with Reprocessing but haven't been able to successfully run the starting project.  I can build to web and native, but trying to hot reload on web is causing issues.  When I run `yarn start` I get the following error: 

```
Fatal error: exception Unix.Unix_error(21, "create_process", "node_modules\\.bin\\bsb")
error Command failed with exit code 2.
```

Trying to run `npm start` results in a longer log file I can post if necessary.

It seems like if I run `python -m SimpleHTTPServer` for web or `yarn start:native` I can still see the default output (blue square), but nothing I change seems to update it after restarting.  I have a feeling that part might be user error, but I'm not sure.  Besides `start` installing the dependencies and building the project seemed to go smoothly.

**Comment by bsansouci** [comment]
Hey! I think I know what it is... On Windows, the file to run `bsb-native` isn't called `bsb` it's called something like `bsb.cmd`. Could you run `dir node_modules\.bin` to see what the file's actually called?

Then to test this, can you modify [this line](https://github.com/Schmavery/reprocessing/blob/1ce7c5373acd1ad3c2c92e7fac3bfe4710bda182/src/bytecode/Reprocessing_Hotreload_Bytecode.re#L44) in `node_modules/reprocessing/src/bytecode/Reprocessing_Hotreload_Bytecode.re` to end with whatever the file name you see is. Then you should maybe add `-clean-world` to the `build` nom script in the `package.json`.

**Comment by mjoerussell** [comment]
`bsb` and `bsb.cmd` both exist in `node_modules\.bin`.  I don't think this is exactly what you were asking for but I changed [this line](https://github.com/Schmavery/reprocessing/blob/1ce7c5373acd1ad3c2c92e7fac3bfe4710bda182/src/bytecode/Reprocessing_Hotreload_Bytecode.re#L28) to 

```reason
let bsb = "node_modules" +/ ".bin" +/ "bsb.cmd";
```

Then I ran `bsb -clean-world` and then rebuilt on web and native.  Hot reloading is now working perfectly on native but not when I run a local server.  This is a great start though!  I'm happy I can start working.
